integration/docker/run_test.go

```diff
@@ -112,16 +110,23 @@ func createLoopDevice() (string, string, error) {
 	}
 
 	// create loop device
-	for i := 0; i < losetupMaxTries; i++ {
-		loopPath := fmt.Sprintf("/dev/loop%d", i)
-		losetupCmd := NewCommand("losetup", "-P", loopPath, f.Name())
-		_, _, exitCode := losetupCmd.Run()
-		if exitCode == 0 {
-			return f.Name(), loopPath, nil
-		}
+	losetupCmd := NewCommand("losetup", "-fP", f.Name())
+	if _, stderr, exitCode := losetupCmd.Run(); exitCode != 0 {
+		return "", "", fmt.Errorf("%s", stderr)
 	}
 
-	return "", "", fmt.Errorf("unable to create loop device for disk %s", f.Name())
+	// get loop device path
+	getLoopPath := NewCommand("losetup", "-j", f.Name())
+	stdout, stderr, exitCode := getLoopPath.Run()
+	if exitCode != 0 {
+		return "", "", fmt.Errorf("%s", stderr)
```

Comment:
Nit: I'd add `stdout`, `stderr` and `exitCode` to the error for maximum debugability.